e2e/testcafe-devextreme/tests/editors/checkBox/common.ts

```diff
@@ -1,49 +1,80 @@
-import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+/* eslint-disable no-restricted-syntax */
```

Comment:
Why we need to have this comment in this file?

Reply:
This comment was used on neighbor tests that utilizes for loops inside .before, without it Lint TS would fail.